<!-- 单次定位，会把localized_pose发送给amcl的初始位姿 -->
<launch>
	<node name="global_localization" pkg="gki_global_localization" type="global_localize_once" output="screen">
		<param name="laser_sigma_hit" value="0.2"/>
		<param name="laser_z_rand" value="0.05"/>
		<param name="laser_z_hit" value="0.95"/>
		<param name="laser_max_range" value="8.0"/>
		<param name="base_frame_id" value="/robot0"/>

		<remap from="scan" to="/robot0/laser_0" />
		<remap from="localized_pose" to="initialpose" />
	</node>
</launch>
